runtime/master/src/main/java/edu/snu/nemo/runtime/master/scheduler/PendingTaskGroupQueue.java

```diff
@@ -36,16 +38,25 @@
 
   /**
    * Enqueues a TaskGroup to this PQ.
-   * @param scheduledTaskGroup to enqueue.
+   * @param scheduledTaskGroup to add.
    */
-  void enqueue(final ScheduledTaskGroup scheduledTaskGroup);
+  void add(final ScheduledTaskGroup scheduledTaskGroup);
```

Comment:
Let's change the name of this class to represent it's behavior well. (It is not a queue anymore.)